viam_cartographer.go

```diff
@@ -326,7 +287,11 @@ func checkIfIMUAndOdometerSupported(ctx context.Context, deps resource.Dependenc
 	// A movement sensor used as an odometer must support Position and Orientation.
 	odometerSupported := properties.PositionSupported && properties.OrientationSupported
 
-	return imuSupported, odometerSupported, nil
+	if !imuSupported && !odometerSupported {
+		return s.ErrMovementSensorNeitherIMUNorOdometer
+	}
+
+	return nil
```

Comment:
@tessavitabile @jeremyrhyde I changed the implementation such that we're now failing to start up the slam service if the provided movement sensor does not support at least an IMU or odometer. Is that ok?

Reply:
Yes